ivy/functional/frontends/paddle/nn/functional/activation.py

```diff
@@ -53,3 +53,14 @@ def hardtanh(
     less = ivy.where(ivy.less(x, min), min, x)
     ret = ivy.where(ivy.greater(x, max), max, less).astype(x.dtype)
     return ret
+
+
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
```

Comment:
Can we please check whether these are actually the only invalid dtypes for this function?

Reply:
![image](https://github.com/unifyai/ivy/assets/86348193/d24b829a-7c8d-4f47-9b7c-4d6c10b29cc3)
Hi @ZiadAmerr , I referred [paddle official docs](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/softsign_en.html) and there it's mentioned, only these 2 dtypes are valid for softsign.